### PR TITLE
adding inline policy for cspm role

### PR DESCRIPTION
**Pull request by sameer-in**
- adding inline policy for cspm role

### CODE REVIEW

File: modules/services/event-bridge/variables.tf [hunk 1]
```diff
@@ -15,7 +15,7 @@ variable "deploy_global_resources" {
   default     = false
 }
 
-variable "organization_units" {
+variable "org_units" {
```
Comment (nkraemer-sysdig) [inline modules/services/event-bridge/variables.tf:18]: This will break the snippets in onboarding-api won't it?

Comment (sameer-in) [reply modules/services/event-bridge/variables.tf:18]: will create a FE ticket to change this.

Comment (haresh-suresh) [reply modules/services/event-bridge/variables.tf:18]: I can go ahead and add it my PR : https://github.com/draios/secure-backend/pull/16067/files

Comment (haresh-suresh) [reply modules/services/event-bridge/variables.tf:18]: @sameer-in But we need to make sure this is merged and published first, then I can merge the BE PR for release

**Review by haresh-suresh — APPROVED** [review]

File: modules/services/event-bridge/variables.tf (hunk 1 above)
Comment (sameer-in) [reply modules/services/event-bridge/variables.tf:18]: FE has pushed the change. so we are safe to merge.

**Review by haresh-suresh — APPROVED** [review]

**Review by nkraemer-sysdig — APPROVED** [review]

**Pull request merged**